axis_backend/apps/kpis/services/kpi_type_service.py

```
@@ -0,0 +1,226 @@
+"""Service for KPIType business logic."""
+from typing import Optional, List, Dict, Any
```

Ruff (0.14.5)

`typing.List` is deprecated, use `list` instead

(UP035)